src/Microsoft.DocAsCode.Common/Git/GitUtility.cs

```diff
@@ -55,45 +57,15 @@ public static GitDetail TryGetFileDetail(string filePath)
 
         public static GitDetail GetFileDetail(string filePath)
         {
-            if (string.IsNullOrEmpty(filePath))
+            if (string.IsNullOrEmpty(filePath) || !ExistGitCommand())
```

Comment:
>ExistGitCommand [](start = 51, length = 15)

Do we need leave a message to tell user reason why the git information doesn't be generated?

Reply:
Yes, https://github.com/dotnet/docfx/pull/2086 does